apps/meteor/app/cloud/server/functions/disconnectWorkspace.ts

```diff
@@ -1,17 +1,4 @@
-import { Settings } from '@rocket.chat/models';
-
-import { retrieveRegistrationStatus } from './retrieveRegistrationStatus';
-import { syncWorkspace } from './syncWorkspace';
-
+/** @deprecated */
 export async function disconnectWorkspace() {
-	const { connectToCloud } = await retrieveRegistrationStatus();
-	if (!connectToCloud) {
-		return true;
-	}
-
-	await Settings.updateValueById('Register_Server', false);
-
-	await syncWorkspace(true);
-
-	return true;
+	throw new Error('Deprecated');
```

Comment:
any special reason to keep this function around? it is an internal function, IMO it is better to remove it and properly assess what it was supposed to do.